lib/react_client.dart

```diff
@@ -148,10 +161,12 @@ class ReactDartComponentFactoryProxy2<TComponent extends Component2> extends Rea
   ReactDartComponentFactoryProxy2(ReactClass reactClass)
       : this.reactClass = reactClass,
         this.reactComponentFactory = React.createFactory(reactClass),
-        this.defaultProps = new JsBackedMap.fromJs(reactClass.defaultProps);
+        this.defaultProps = new JsBackedMap.fromJs(reactClass?.defaultProps);
```

Comment:
if someone passes in `null`? lol ill remove this i think this was left over from before when i was considering just using `ReactDartComponentFactoryProxy2` instead of creating another factory.